pkg/console/operator/operator.go

```diff
@@ -123,37 +135,31 @@ func NewConsoleOperator(
 		// console resources
 		// core kube
 		secretsClient:    corev1Client,
+		secretsLister:    secretsInformer.Lister(),
 		configMapClient:  corev1Client,
 		serviceClient:    corev1Client,
 		nodeClient:       corev1Client,
 		deploymentClient: deploymentClient,
 		dynamicClient:    dynamicClient,
 		// openshift
-		routeClient:   routev1Client,
-		oauthClient:   oauthv1Client,
-		versionGetter: versionGetter,
+		oauthClientLister: oauthClientInformer.Lister(),
+		routeClient:       routev1Client,
+		routeLister:       routeInformer.Lister(),
+		versionGetter:     versionGetter,
 		// plugins
 		consolePluginLister: consolePluginInformer.Lister(),
 
 		resourceSyncer: resourceSyncer,
 	}
 
-	secretsInformer := coreV1.Secrets()
-	configMapInformer := coreV1.ConfigMaps()
-	managedConfigMapInformer := managedCoreV1.ConfigMaps()
-	serviceInformer := coreV1.Services()
-	nodeInformer := coreV1.Nodes()
-	configV1Informers := configInformer.Config().V1()
-	configNameFilter := util.IncludeNamesFilter(api.ConfigResourceName)
-	targetNameFilter := util.IncludeNamesFilter(api.OpenShiftConsoleName)
-
 	informers := []factory.Informer{
 		configV1Informers.Consoles().Informer(),
 		operatorConfigInformer.Informer(),
 		configV1Informers.Infrastructures().Informer(),
 		configV1Informers.Ingresses().Informer(),
 		configV1Informers.Proxies().Informer(),
 		configV1Informers.OAuths().Informer(),
+		oauthClientInformer.Informer(),
```

Comment:
This means we're only watching an OAuthClient named 'cluster'. Is that correct?

Reply:
good catch, I did not notice these informers were actually filtered by name